src/engine.tsx

```diff
@@ -291,15 +291,12 @@ const Engine: {
       Player.gainMoney(offlineHackingIncome, "hacking");
       // Process offline progress
       loadAllRunningScripts(Player); // This also takes care of offline production for those scripts
+      Player.focus = true;
```

Comment:
hmm wont this cause some unintended side effects? or is it set back to false all over the joint?